frontend_lib/test/helper.spec.js

```diff
@@ -150,15 +151,14 @@ describe('helper.js', () => {
       username: 'username'
     }
     const objectToSerialize = {
-      avatar_url: undefined,
       email: null,
       user_id: 0,
-      public_name: 'a_name',
+      public_name: '',
       lang: 'pt',
-      username: ''
+      username: undefined
```

Comment:
I understand that you want to test that the username key will be present after serialization (even if `undefined`).
Then the test bellow is not sufficient. You need to test `Object.prototype.hasOwnProperty.call(serializedObj, 'username')` or something like that.

If you don't care whether the key is actually present, feel free to ignore this comment and resolve the conversation :-)